src/gtc/gtir.py

```diff
@@ -270,3 +269,4 @@ def param_names(self) -> List:
         return [p.name for p in self.params]
 
     _validate_symbol_refs = common.validate_symbol_refs()
+    _validate_lvalue_dims = common.validate_lvalue_dims()
```

Comment:
We need to define at some point (after upcoming eve changes), how much of this root-node validations we want to run automatically. For now, it's good, I'd say.